game-core/src/main/java/games/strategy/engine/data/export/GameDataExporter.java

```diff
@@ -163,8 +163,8 @@ private void propertyList(final GameData data) { // TODO: Unchecked Reflection
   }
 
   private void printEditableProperties(final Map<String, IEditableProperty> edProperties) {
-    for (final String property : edProperties.keySet()) {
-      printEditableProperty(edProperties.get(property));
+    for (final Map.Entry<String, IEditableProperty> stringIEditablePropertyEntry : edProperties.entrySet()) {
+      printEditableProperty(stringIEditablePropertyEntry.getValue());
```

Comment:
The key doesn't appear to be used here.  Could we just iterate over `edProperties.values()`?